nova/manifests/compute.pp

```diff
@@ -131,6 +131,9 @@
   $internal_service_availability_zone = 'internal',
   $heal_instance_info_cache_interval  = '60',
   $pci_passthrough                    = undef,
+  $auth_protocol                      = 'http',
+  $auth_uri                           = 'http://127.0.0.1:5000/v2.0',
+  $identity_uri                       = 'http://127.0.0.1:35357',
```

Comment:
I'm confused---these are Keystone ports, so I'm assuming that this is for nova-compute's API calls to Keystone.  Why are we using HTTP here, and why is the address localhost?